projects/kubernetes/autoscaler/1-23/helm/schema.json

```diff
@@ -0,0 +1,151 @@
+{
+  "affinity": {
+  },
+  "autoDiscovery": {
+    "clusterName": null,
+    "tags": [
+      "k8s.io/cluster-autoscaler/enabled",
+      "k8s.io/cluster-autoscaler/{{ .Values.autoDiscovery.clusterName }}"
+    ],
+    "roles": [
+      "worker"
+    ],
+    "labels": [
+
+    ]
+  },
+  "autoscalingGroups": [
+
+  ],
+  "autoscalingGroupsnamePrefix": [
+
+  ],
+  "clusterAPIMode": "incluster-incluster",
+  "clusterAPIKubeconfigSecret": "",
+  "clusterAPIWorkloadKubeconfigPath": "/etc/kubernetes/value",
+  "clusterAPICloudConfigPath": "/etc/kubernetes/mgmt-kubeconfig",
+  "clusterAPIConfigMapsNamespace": "",
+  "cloudConfigPath": "",
+  "cloudProvider": "clusterapi",
+  "containerSecurityContext": {
+  },
+  "deployment": {
```

Comment:
For now I'm leaving all of these fields exposed while I test if that's ok. I expect that I will be able to strip down a lot of them before release.

That said, it seems to me we would rather err on the permissive than restrictive side. Does that align with packages team experience?

Reply:
yes